ast/builtins.go

```diff
@@ -2289,6 +2290,17 @@ var CryptoX509ParseCertificateRequest = &Builtin{
 	),
 }
 
+var CryptoX509ParseKeyPair = &Builtin{
+	Name:        "crypto.x509.parse_keypair",
+	Description: "Returns a valid key pair",
+	Decl: types.NewFunction(
+		types.Args(
+			types.Named("cert", types.S).Description("string containing PEM or base64 encoded DER certificates"),
+			types.Named("pem", types.S).Description("string containing PEM or base64 encoded private key"),
+		),
+		types.Named("output", types.NewObject(nil, types.NewDynamicProperty(types.S, types.A))).Description("if key pair is valid, returns the tls.certificate(https://pkg.go.dev/crypto/tls#Certificate) as an object. The object contains the fields certificate data, PrivateKey, SupportedSignatureAlgorithms, OCSPStaple, SignedCertificateTimestamps as well as the parsed leaf certificate. If the key pair is invalid, nil and an error are returned."),
+	),
```

Comment:
Per the description of `tls.X509KeyPair` the parsed leaf certificate will be nil. Do we need to update the description?
```
X509KeyPair parses a public/private key pair from a pair of PEM encoded data. On successful return, Certificate.Leaf will be nil because the parsed form of the certificate is not retained.
```